composer.json

```diff
@@ -32,20 +32,20 @@
         "kalnoy/nestedset": "^6.0",
         "khaled.alshamaa/ar-php": "^6.0.0",
```

Comment:
This dependency now needs `ext-calender` also. Please add `ext-calender`  to our `composer.json` file also.